docs/reference/release-notes/v2.28/index.md

```diff
@@ -0,0 +1,121 @@
+  // your existing config
+  flags: {
+    FAST_DEV: true,
+  },
+}
+```
+
+Currently supported flags:
+
+- `DEV_SSR` - Server-side render (SSR) pages on full reloads during develop. Helps you detect SSR bugs and fix them without needing to do full builds.
+- `QUERY_ON_DEMAND` - Only run queries when needed instead of running all queries upfront. Speeds starting the develop server.
+- `LAZY_IMAGES` - Don't process images during development until they're requested from the browser. Speeds starting the develop server. See [Experimental: Lazy images in develo](#experimental-lazy-images-in-develop) section for additional details.
+- `FAST_DEV` - Umbrella flag that enables `DEV_SSR`, `QUERY_ON_DEMAND` and `LAZY_IMAGES` features.
+
+## Removed experimental lazy page bundling
+
+In gatsby@2.27.0 we added [Experimental: Lazy page bundling](../v2.27/index.md#experimental-lazy-page-bundling-in-development) mode for `gatsby develop` that would delay compiling page templates until it was needed. While preliminary tests were very promising, we discovered few showstoppers that degraded development experience. [We decided to end the experiment](https://github.com/gatsbyjs/gatsby/discussions/28137#discussioncomment-138998) for now and shift our efforts to [Less aggressive cache invalidation](#less-aggressive-cache-invalidation).
+
+## Making `gatsby develop` faster
+
+We continue working on speeding up development server startup:
+
+### Experimental: Lazy images in develop
+
+As more and more images are added to a Gatsby site, the slower the local development experience oftentimes becomes.
+You spend time waiting for images to process, instead of you know, developing! No longer!
+This experimental version of `gatsby-plugin-sharp` only does image processing when the page gets requested.
+
+To use it, make sure you have `gatsby-plugin-sharp@^2.10.0` and add `LAZY_IMAGES` to flags in `gatsby-config.js`:
+
+```js
+// In your gatsby-config.js
+module.exports = {
+  // your existing config
+  flags: {
+    LAZY_IMAGES: true,
+  },
+}
+```
+
+[Details and discussion](https://github.com/gatsbyjs/gatsby/discussions/27603).
+
+## Notable bugfixes
+
+...
+
+## Work in progress
+
+### Less aggressive cache invalidation
+
+Gatsby aggressively clears its cache, sometimes too aggressively. Here's a few examples:
+
+- You `npm install` a plugin, or update an existing
+- You change your `gatsby-node.js` and add a few log statements
+- You change your `siteMetadata` in `gatsby-config.js` to update your site's title
+
+In all of these caches, your cache is entirely cleared, which means that the next time you run `gatsby develop` the experience is slower than it needs to be. We'll be working on this to ensure that your first run, and every run thereafter, is as quick and speedy as you expect!
+
+
+
+## Contributors
+
+A big **Thank You** to [everyone who contributed](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0) to this release 💜
```

Comment:
Maybe give the count + list names esp community members? Plus the length of time since 2.27